### ISSUE TITLE
Defining a class or enum without namespace will cause codegen to fail

### ISSUE THREAD

**Issue by jschieck** [dotnet/orleans#369]
If you do not specify a namespace for a class or enum and try and reference it in an IGrainState, codegen with fail by putting in:
```dotnet
using ;
```
in the serialization class, and you'll get an "Identifier expected" error message.

Example:

```dotnet
// data class
namespace MyNamespace
{
	public class PlayerData
	{
		public MyEnum enumVal { get; set; }
	}
}

// state
namespace GrainInterfaces
{
    public interface IPlayerState : IGrainState
    {
        PlayerData playerData { get; set; }
    }
}

// in another class, with no namespace
public enum MyEnum
{
	Dead,
	Alive,	
}
```

**Comment by gabikliot** [comment]
Thanks for pointing that out. We will take a look.

**Comment by gabikliot** [comment]
Fixed via #374.
